cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -1388,6 +1431,14 @@ TEST_F(TestLargeBinaryParquetIO, Basics) {
   const auto arrow_properties =
       ::parquet::ArrowWriterProperties::Builder().store_schema()->build();
   this->RoundTripSingleColumn(large_array, large_array, arrow_properties);
+
+  ArrowReaderProperties arrow_reader_properties;
+  arrow_reader_properties.set_use_large_binary_variants(true);
+  // Input is narrow array, but expected output is large array, opposite of the above tests.
+  // This validates narrow arrays can be read as large arrays.
+  this->RoundTripSingleColumn(narrow_array, large_array,
+                              default_arrow_writer_properties(),
```

Comment:
If `:parquet::ArrowWriterProperties::Builder().store_schema()` is set, what do we return in the current impl? large_array right?

Reply:
Yes